### ISSUE TITLE
Music 0.3.7 does not work with new Ampache iOS app

### ISSUE THREAD

**Issue by qirtaiba** [owncloud/music#406]
I am using the new Ampache iOS app, which appears to be a rewritten version of AmpacheX. I can connect to my ownCloud server fine, and when I sync it appears to work, ie. does not throw errors. But once it is done I can only see the list of Artists in the iOS app, I can't see any Songs or Albums. In the Artists view, it shows how many albums/songs they have, but I can't tap through to play them. I tried to resync (left it doing that overnight), but it didn't help.

**Comment by MorrisJobke** [comment]
@qirtaiba Is there anything in the ownCloud.log file in your data directory?

**Comment by qirtaiba** [comment]
Yes, 1.1Gb of stuff, but by stripping out what seems irrelevant I'm left with this:

> {"reqId":"kmc2VDZfKdaeILpMaQnJ","remoteAddr":"","app":"PHP","message":"Class 'OCA\\Music\\DB\\AmpacheSessionMapper' not found at \/var\/www\/owncloud\/apps\/music\/app\/music.php#172","level":3,"time":"2015-07-28T03:33:33+00:00","method":"--","url":"--"}
> {"reqId":"PnoVT9YU87Dj07SYinPm","remoteAddr":"192.168.42.145","app":"index","message":"Exception: {\"Exception\":\"OC\\\\NeedsUpdateException\",\"Message\":\"\",\"Code\":0,\"Trace\":\"#0 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/app.php(99): OC_App::loadApp('music')\\n#1 \\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/util.php(127): OC_App::loadApps(Array)\\n#2 \\\/var\\\/www\\\/owncloud\\\/lib\\\/base.php(866): OC_Util::setupFS()\\n#3 \\\/var\\\/www\\\/owncloud\\\/index.php(40): OC::handleRequest()\\n#4 {main}\",\"File\":\"\\\/var\\\/www\\\/owncloud\\\/lib\\\/private\\\/app.php\",\"Line\":118}","level":4,"time":"2015-07-28T03:47:40+00:00","method":"GET","url":"\/owncloud\/index.php\/core\/js\/oc.js?v=95849ed8ae2c789c010a577edb916fe8"}
> {"reqId":"ax3B7B0k2eWDZDJrSMxn","remoteAddr":"192.168.42.145","app":"music","message":"JS: Processed more files than available 10819\/0","level":0,"time":"2015-07-28T03:51:27+00:00","method":"POST","url":"\/owncloud\/index.php\/apps\/music\/api\/log"}
> {"reqId":"w6LRGmMT49NBmSVmPAo1","remoteAddr":"192.168.42.145","app":"music","message":"JS: Processed more files than available 10819\/0","level":0,"time":"2015-07-28T04:08:32+00:00","method":"POST","url":"\/owncloud\/index.php\/apps\/music\/api\/log"}
> {"reqId":"c6yoM8Tz5VZoNFzsVzUZ","remoteAddr":"192.168.42.145","app":"music","message":"JS: Processed more files than available 10819\/0","level":0,"time":"2015-08-03T05:40:13+00:00","method":"POST","url":"\/owncloud\/index.php\/apps\/music\/api\/log"}
> {"reqId":"oq6BoeA9KhiXrhrtV6Hx","remoteAddr":"208.90.213.162","app":"music","message":"JS: Processed more files than available 10819\/0","level":0,"time":"2015-08-03T16:33:46+00:00","method":"POST","url":"\/owncloud\/index.php\/apps\/music\/api\/log"}

**Comment by MorrisJobke** [comment]
@qirtaiba Looks weird. Can I ask you to restart apache (and php-fpm if you use it) to clear eventually cached opcode?

**Comment by qirtaiba** [comment]
On 6/08/2015 12:54 am, Morris Jobke wrote:
>
> @qirtaiba <https://github.com/qirtaiba> Looks weird. Can I ask you to
> restart apache (and php-fpm if you use it) to clear eventually cached
> opcode?
>

There's no php-fpm, it's just an ordinary Apache with mod-php.  I
restarted it now.

-- 
Jeremy Malcolm PhD LLB (Hons) B Com
Internet lawyer, ICT policy advocate, geek
echo "9EEAi^^;6C6]>J^=^>6"|tr '\!-~' 'P-~\!-O'|wget -q -i - -O -

**Comment by MobileManiC** [comment]
I can also confirm that behavior. Other clients like Tomahawk on Mac is working OK for me.

**Comment by paulijar** [comment]
So much has changed in the Music app and its Ampache support during the past 6 years that I doubt that this old bug is relevant any more. If similar problems still occur, then a new issue should be opened.
